src/app/static/webpack.config.js

```diff
@@ -36,6 +36,7 @@ module.exports = {
     ],
     output: {filename: '[name].bundle.js', path: path.resolve(__dirname, 'public/js')},
     resolve: {
+        extensions: ['*', '.js', '.vue', '.json'],
```

Comment:
This means that you can write 
`import Component from component`
instead of `import Component from component.vue`